src/Bitstream-Vera.xml

```diff
@@ -0,0 +1,16 @@
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license"><license isOsiApproved="false" licenseId="Bitstream-Vera" listVersionAdded="3.17" name="Bitstream Vera Font License">
+    
+    <crossRefs><crossRef>https://web.archive.org/web/20080207013128/http://www.gnome.org/fonts/</crossRef><crossRef>https://docubrain.com/sites/default/files/licenses/bitstream-vera.html</crossRef></crossRefs><standardLicenseHeader>none</standardLicenseHeader><notes />
+    
+    <text><p>Copyright
+Copyright (c) 2003 by Bitstream, Inc. All Rights Reserved. Bitstream Vera is a trademark of Bitstream, Inc.</p>
+        <p>Permission is hereby granted, free of charge, to any person obtaining a copy of the fonts accompanying this license (&#8220;Fonts&#8221;) and associated documentation files (the &#8220;Font Software&#8221;), to reproduce and distribute the Font Software, including without limitation the rights to use, copy, merge, publish, distribute, and/or sell copies of the Font Software, and to permit persons to whom the Font Software is furnished to do so, subject to the following conditions:</p>
```

Comment:
Here and below, it looks like this is doing an encoding of "smart quotes" for the defined terms -- `“ ”` instead of `" "`. The test text file also appears to have smart quotes too. Guessing these should probably all be replaced by regular quote marks?

Reply:
I think I've fixed them now